ev-stacks/stacks/fullnode/entrypoint.fullnode.sh

```
@@ -0,0 +1,241 @@
+#!/bin/sh
+# Fail on any error
+set -e
+
+# Fail on any error in a pipeline
+set -o pipefail
```

Shellcheck (0.10.0)

[warning] In POSIX sh, set option pipefail is undefined.

(SC3040)